src/scrape_up/Pixabay/pixabay.py

```diff
@@ -0,0 +1,137 @@
+import requests
+
+
+class Pixabay():
+    """
+    Class - `Pixabay`\n
+
+    | Methods                     | Details                                                                                              |
+    | --------------------------- | ---------------------------------------------------------------------------------------------------- |
+    | `.get_params_video()`       | Returns the list of parameters required to load the videos from Pixaby.                              |
+    | `.get_video()`              | Downloads the videos from pixaby to the local storage.                                               |
+    | `.get_params_photo()`       | Returns the list of parameters required to load pics from Pixaby.                                    |
+    | `.get_photo()`              | Downloads the photos from pixaby to local storage.                                                   |
+
+
+    """
+    def __init__(self, api_key='38504833-19606430bd8fde504120d1630', verbose=True):
+        self.api_key = api_key
```

Comment:
If you have hardcoded the API key, make sure to make it private, so that others can not access the key. Also, make helpers function like `_get_params_video` private so that we can allow the user to access the only two main methods we want.

Reply:
Hey @nikhil25803 I've implemented the proposed changes. Please review!